conans/test/integration/command/test_profile.py

```diff
@@ -1,7 +1,31 @@
+import os 
+
 from conans.test.utils.tools import TestClient
 
 
 def test_profile_path():
     c = TestClient()
     c.run("profile path default")
     assert "default" in c.out
+
+
+def test_ignore_paths_when_listing_profiles():
+    c = TestClient()
+    ignore_paths = ['.DS_Store']
+
+    # Create files, dirs if necessary, at the ignore paths
+    for path in ignore_paths:
+        rel_file_path = os.path.join(c.cache.profiles_path, path)
+
+        base_dir = os.path.join(c.cache.profiles_path, os.path.dirname(path))
+        if base_dir:
+            os.makedirs(base_dir, exist_ok=True)
+            open(os.path.join(base_dir, os.path.basename(path)), 'w').close()
+        else:
+            open(rel_file_path, 'w').close()
```

Comment:
I'd say this is a bit overly complex. Something like:
```python
os.makedirs(c.cache.profiles_path, exist_ok=True)  # just in case
os.utime(os.path.join(c.cache.profiles_path, ".DS_Store"))  # This is a "touch"
```
should be good enough

Reply:
This still needs `open(os.path.join(c.cache.profiles_path, '.DS_Store')).close()' to work I think. As far as I am aware `os.utime` just updates the last modified time of the file.

Reply:
Yes, you are right, I thought that ``utime`` was equivalent to ``touch`` and create the file if not existing.
Yes, the ``open()`` might be necessary then. There is some ``save(path, content)`` helper function too, but the open() is fine.